test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/CaseSensitiveTagHelperAttributeComparer.cs

```diff
@@ -22,16 +21,13 @@ public bool Equals(IReadOnlyTagHelperAttribute attributeX, IReadOnlyTagHelperAtt
                 attributeX == attributeY ||
                 // Normal comparer doesn't care about the Name case, in tests we do.
                 string.Equals(attributeX.Name, attributeY.Name, StringComparison.Ordinal) &&
-                Equals(attributeX.Value, attributeY.Value);
+                Equals(attributeX.Value, attributeY.Value) &&
+                attributeX.Minimized == attributeY.Minimized;
         }
 
         public int GetHashCode(IReadOnlyTagHelperAttribute attribute)
         {
-            return HashCodeCombiner
-                .Start()
-                .Add(attribute.Name, StringComparer.Ordinal)
-                .Add(attribute.Value)
-                .CombinedHash;
+            return attribute.GetHashCode();
```

Comment:
:+1: though I don't see how this relates to the rest of this PR.  should at least be separately mentioned in the PR and commit descriptions.